app/spec/models/post_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe '.recently' do
+    subject { described_class.recently }
+
+    let!(:post1) { create(:post, posted_at: DateTime.new(2024, 9, 1, 9, 0, 1)) }
+    let!(:post2) { create(:post, posted_at: DateTime.new(2024, 9, 1, 9, 0, 2)) }
+    let!(:post3) { create(:post, posted_at: DateTime.new(2024, 9, 1, 8, 0, 0)) }
```

Comment:
DateTime じゃなくて TimeWithZone を使うようにしたいですね

```suggestion
    let!(:post1) { create(:post, posted_at: '2024-09-01 08:00:00'.in_time_zone)) }
```

または、単に文字列を渡して ActiveRecord のキャストに任せるとか

```suggestion
    let!(:post3) { create(:post, posted_at: '2024-09-01 08:00:00')) }
```